webrtc-charter.html

```diff
@@ -320,8 +326,7 @@ <h3>
 <li>Feature independence: Features may be introduced in the new APIs that are not available when using the PeerConnection API.</li>
 </ul>
 
-<p>A use-case document will be developed to clarify the functionality that the new APIs are to provide.</p>				
-<p>The Working Group will take the work done by the ORTC Community Group as a source of input, and when contemplating similar APIs in the Working Group, make efforts to align with the ORTC CG on API methodologies and nomenclature. This may include scheduled design meetings with relevant WG and CG stakeholders to foster convergence of the APIs.</p>
+<p>A <a href="https://www.w3.org/TR/webrtc-nv-use-cases/">use-case document</a> is maintained  to identify the functionality that the new APIs are to provide.</p>
```

Comment:
would phrase this as "to identify the functionality that needs new APIs for realization".